llvm/lib/Transforms/IPO/AttributorAttributes.cpp

```diff
@@ -5187,6 +5187,59 @@ struct AADereferenceableCallSiteReturned final
 // ------------------------ Align Argument Attribute ------------------------
 
 namespace {
+
+Align getKnownAlignForIntrinsic(Attributor &A, AAAlign &QueryingAA,
+                                const IntrinsicInst &II) {
+  switch (II.getIntrinsicID()) {
+  case Intrinsic::ptrmask: {
+    const auto *ConstVals = A.getAAFor<AAPotentialConstantValues>(
+        QueryingAA, IRPosition::value(*(II.getOperand(1))), DepClassTy::NONE);
+    const auto *AlignAA = A.getAAFor<AAAlign>(QueryingAA, IRPosition::value(II),
```

Comment:
I don't think you really need to pull `AAAlign` for the value here, because if you set `TrackUse` to `true`, after you inline this function to its caller, its MBE uses will be checked automatically by existing code.